### ISSUE TITLE
The navbar width becomes messed up upon clicking on my profile picture

### ISSUE THREAD

**Issue by tristan957** [thepracticaldev/dev.to#1160]
**Describe the bug**
The overlay seems to be cutting into the navbar.

![image](https://user-images.githubusercontent.com/11936772/48749555-1d18e400-ec41-11e8-9e49-6abdbae59832.png)

**To Reproduce**

1. Click profile picture

**Expected behavior**
Navbar to be above overlay.

**Desktop (please complete the following information):**

- OS: Solus (Linux)
- Browser Firefox
- Version 63.0.1

**Comment by Link2Twenty** [comment]
`.menubg` starts too high, it has a top of 40px whereas the `.top-bar` has a height of 48px. A change in this block ought to fix it.

https://github.com/thepracticaldev/dev.to/blob/f744f43493ef4f90f5421665d0d393db03da052d/app/assets/stylesheets/top-bar.scss#L204-L213

**Comment by Link2Twenty** [comment]
Though personally, rather than having to worry about sizes I would put the model effect over the `page-content`

perhaps something like this

```CSS
div#page-content::after {
    content: "";
    position: fixed;
    top: 0;
    left: 0;
    height: 100vh;
    width: 100vw;
    background: rgba(17, 17, 17, 0.88);
    z-index: 999;
    opacity: 0;
    pointer-events: none;
}

div#page-content.active-modal::after {
    opacity: 1;
    pointer-events: auto;
}
```
This would mean the whole of `.top-bar` is above the modal backdrop, also anything could use that modal style in the future.

**Side note:** 
I'm using `opacity` and `pointer-events` rather than `display:none` to avoid a repaint.

**Comment by jessleenyc** [comment]
Thanks for flagging! This issue is approved for anyone who might want to work on it.

**Comment by Link2Twenty** [comment]
This issue can be closed now

**Comment by maestromac** [comment]
Resolved with #1227 . Thanks @Link2Twenty !
